.gitignore

```diff
@@ -148,3 +148,4 @@ docker-compose.yml
 # Documentation
 /docs/build
 
+
```

Comment:
?

Reply:
I thought an extra space would be nice, will be fixed in the next commit.